### PR TITLE
[WIP] Implement the quick start option (aka mocked start)

### PR DESCRIPTION
**Pull request by zah**
For more info, see here:
https://github.com/ethereum/eth2.0-pm/tree/master/interop/mocked_start

### CODE REVIEW

**Comment by zah** [comment]
This needs a small bug fix in Confutils, so I'll probably merge it after I get back from vacation at the end of next week.

File: beacon_chain/quickstart.nim [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    copyMem(addr paddedBytes[0], addr leBytes[0], sizeof(leBytes))
+    var hash = eth2hash(paddedBytes)
+    validator.privKey.initFromBytes hash.data
+    validator.pubKey = validator.privKey.pubKey
+
+    result.add validator
+
+func quickStartDeposits*(validators: openarray[ValidatorKeyPair]): seq[Deposit] =
+  for validator in validators:
```
Comment (arnetheduck) [inline beacon_chain/quickstart.nim]: so these are not just for quickstart deposits, but any, being part of the deposit contract: https://github.com/ethereum/eth2.0-specs/blob/dev/specs/core/0_deposit-contract.md#withdrawal-credentials

File: beacon_chain/quickstart.nim [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import
+  stint, stew/endians2,
+  spec/[crypto, digest, datatypes], conf, ssz
+
+func quickStartValidators*(n: uint64): seq[ValidatorKeyPair] =
```
Comment (arnetheduck) [inline beacon_chain/quickstart.nim]: tests?

**Comment by zah** [comment]
After https://github.com/status-im/nim-beacon-chain/pull/507, this is no longer relevant.

**Pull request closed without merging**